game-core/src/test/java/games/strategy/triplea/delegate/MoveValidatorTest.java

```diff
@@ -58,7 +57,7 @@ void testGetLeastMovement() {
     final Collection<Unit> collection = bomber.create(1, british);
     assertEquals(new BigDecimal(6), MoveValidator.getLeastMovement(collection));
     final Object[] objs = collection.toArray();
-    ((TripleAUnit) objs[0]).setAlreadyMoved(BigDecimal.ONE);
+    ((Unit) objs[0]).setAlreadyMoved(BigDecimal.ONE);
```

Comment:
```suggestion
    collection.iterator().next().setAlreadyMoved(BigDecimal.ONE);
```

And remove the line above

Reply:
Going to shy away from this update, there are many more updates we can do. This line is only swapping the casted type, I do not want to do more than just that (it opens up the flood gates, in which case such a giant is unlikely to be accurate, ever be finished, and would not be reviewable)